virttest/test_setup.py

```diff
@@ -2374,16 +2374,21 @@ def enable(self):
         if not self.log_file or not os.path.isdir(os.path.dirname(self.log_file)):
             self.log_file = utils_misc.get_path(self.test.debugdir,
                                                 "libvirtd.log")
+        if os.path.isfile(self.backupfile):
+            os.remove(self.backupfile)
+
+        # backup libvirtd conf before restarting libvirtd
+        fd = open(self.backupfile, "w")
```

Comment:
Maybe use `with` statement this time.

Reply:
sure, agree. I would make the change as suggested.